codespell_lib/_codespell.py

```diff
@@ -232,6 +232,10 @@ def parse_options(args):
                         help='Comma separated list of words to be ignored '
                              'by codespell. Words are case sensitive based on '
                              'how they are written in the dictionary file')
+    parser.add_argument('-P', '--sub-pairs', type=str, metavar='FILE',
+                        help='Custom substitution text file that contains '
+                             'substituions key value pairs.  Can be used to '
```

Comment:
substituions->substitutions

Reply:
Funny, because this made it through the codespell check.